src/mobile_crawler/core/crawler_loop.py

```
@@ -1120,25 +1134,44 @@ def _initialize_video_recording(self, run_id: int, session_path: str) -> None:
-    def _stop_video_recording(self) -> None:
-        """Stop video recording and save video file."""
+    def _stop_video_recording(self, run_id: int) -> None:
+        """Stop video recording and save video file.
+        
+        Args:
+            run_id: Run ID for event logging
+        """
+        logger.debug(f"[DEBUG] _stop_video_recording called for run {run_id}")
+        
         if not self._video_recording_manager:
+            logger.debug(f"[DEBUG] _video_recording_manager is None in _stop_video_recording")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)